openedx/features/student_certificates/helpers.py

```diff
@@ -74,6 +74,17 @@ def get_certificate_image_url_by_uuid(verify_uuid):
     )
 
 
+def get_course_display_name_by_uuid(verify_uuid):
+    """
+    :param certificate uuid:
+    :return: display name of the course
+    """
+    course_id = GeneratedCertificate.objects.get(verify_uuid=verify_uuid).course_id
```

Comment:
What if certificate doesn't exist against provided **verify_uuid**?
Please handle it properly.